### DISCUSSION TITLE
How to add custom dataset exported from Roboflow using semantic segmentation masks format ?

### DISCUSSION THREAD

**Discussion by saeedadeeb103** [open-mmlab/mmengine#1253] [Q&A] [answered]
I annotated my dataset using Roboflow and  I exported it using semantic segmentation masks format but I didnt know how to use it in mmseg, i would thankful if someone knows how to do it.

**Comment by zhouzaida** [comment, marked as answer]
Hi, you can open an issue in https://github.com/open-mmlab/mmsegmentation/issues.

**Reply by saeedadeeb103** [reply]
noted
